src/components/EpicDetailView/EpicDetailView.tsx

```diff
@@ -38,10 +43,29 @@ export function EpicDetailView({
   created: string
   updated: string
   closeModal: () => void
+  startDate: Date
+  dueDate: Date
 }) {
   const queryClient = useQueryClient()
   const reloadEpics = () => queryClient.invalidateQueries({ queryKey: ["epics"] });
 
+  const useDateMutation = (property: string) => useMutation({
```

Comment:
Is it intentional for the epic's start date to precede its creation date?

Reply:
Yes, as you may want to retroactively add epics you have done in the past. Or maybe even just create an epic that already started and you currently work on.